src/main/webapp/js/theme.js

```diff
@@ -0,0 +1,42 @@
+// RoyalBlue
+// Default theme
+function theme1() {
+    document.documentElement.style.setProperty('--main-color', '#2e5bdc');
+    document.documentElement.style.setProperty('--main-color-background', 'rgba(46, 91, 220, 0.2)');
+    document.documentElement.style.setProperty('--button-color', '#5a82f1');
+}
+
+// Summer Sky
+function theme2() {
+    document.documentElement.style.setProperty('--main-color', '#38d1f1');
+    document.documentElement.style.setProperty('--main-color-background', 'rgba(56, 209, 241, 0.2)');
+    document.documentElement.style.setProperty('--button-color', '#38d1f1');
+}
+
+// Islamic Green
```

Comment:
can we give this a name without any religious associations? How about fun names like "Peaceful green", "Grass green", "graceful green", etc.

Reply:
sorry! I took the names straight from a hex color website